src/go/rpk/pkg/cli/cmd/acl/common.go

```diff
@@ -0,0 +1,384 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package acl
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/Shopify/sarama"
+	"github.com/hashicorp/go-multierror"
+)
+
+const (
+	resourceFlag       = "resource"
+	resourceNameFlag   = "resource-name"
+	namePatternFlag    = "name-pattern"
+	allowPrincipalFlag = "allow-principal"
+	allowHostFlag      = "allow-host"
+	denyPrincipalFlag  = "deny-principal"
+	denyHostFlag       = "deny-host"
+)
+
+// These need to match the ones in Sarama:
+// https://github.com/Shopify/sarama/blob/41df78df10a9ef3c807cbe1f2814001e330fbdf1/acl_types.go#L172-L176
+func supportedResources() []string {
+	return []string{"*", "cluster", "group", "topic", "delegation-token", "transactionalid"}
+}
+
+// These need to match the ones in Sarama:
+// https://github.com/Shopify/sarama/blob/41df78df10a9ef3c807cbe1f2814001e330fbdf1/acl_types.go#L200-L203
+func supportedPatterns() []string {
+	return []string{"any", "match", "literal", "prefixed"}
+}
+
+// These need to match the ones in Sarama:
+// https://github.com/Shopify/sarama/blob/41df78df10a9ef3c807cbe1f2814001e330fbdf1/acl_types.go#L68-L79
```

Comment:
@Mateoc  - if the acls are set on the command line, do they get displayed correctly in yack?